packages/auth/src/grant/routes.test.ts

```diff
@@ -987,4 +935,197 @@ describe('Grant Routes', (): void => {
       })
     })
   })
+
+  describe('/continue', (): void => {
+    test('Can issue access token', async (): Promise<void> => {
+      const grant = await Grant.query().insert({
+        state: GrantState.Granted,
+        startMethod: [StartMethod.Redirect],
+        continueToken: crypto.randomBytes(8).toString('hex').toUpperCase(),
+        continueId: v4(),
+        finishMethod: FinishMethod.Redirect,
+        finishUri: 'https://example.com',
+        clientNonce: crypto.randomBytes(8).toString('hex').toUpperCase(),
+        clientKeyId: KEY_REGISTRY_ORIGIN + TEST_KID_PATH,
+        interactId: v4(),
+        interactRef: v4(),
+        interactNonce: crypto.randomBytes(8).toString('hex').toUpperCase()
+      })
```

Comment:
Is that different from the one you create in `beforeEach`?

Reply:
The difference is that `GrantState` is "Granted" rather than "Pending", will update this to be more clear.